src/globus_sdk/services/transfer/client.py

```diff
@@ -1235,6 +1235,40 @@ def operation_rename(
             query_params=query_params,
         )
 
+    def operation_stat(
+        self,
+        endpoint_id: UUIDLike,
+        path: str | None = None,
+        *,
+        local_user: str | None = None,
+        query_params: dict[str, t.Any] | None = None,
+    ) -> response.GlobusHTTPResponse:
+        """
+        :param endpoint_id: The ID of the collection on which to do the stat operation
+        :param path: Path on the collection to do the stat operation on
+        :param local_user: Optional value passed to identity mapping specifying which
+            local user account to map to. Only usable with Globus Connect Server v5
+            mapped collections.
+        :param query_params: Additional passthrough query parameters
+            .. tab-item:: API Info
```

Comment:
This looks accidentally-truncated. You need the `.. tab-set::` directive above.

I'd also like it if we used the `_testing` data to generate an example response here. I just did this method, so it's what I'll use as an example:
https://github.com/globus/globus-sdk-python/blob/8e06b1b70202ca954c6a343b8bfca617c39d249e/src/globus_sdk/services/groups/client.py#L103-L105
(`expandtestfixture` does the `_testing` lookup and splats the result into a generated code-block)

Maybe another tab with example usage (like `operation_rename`, above) would be good to add as well.